addon/index.js

```diff
@@ -32,6 +32,33 @@ function maybeUnwrapProxy(o) {
   return isProxy(o) ? maybeUnwrapProxy(safeGet(o, 'content')) : o;
 }
 
+function getContent(ctx) {
+  return ctx._content ? ctx._content : ctx.content ? ctx.content : ctx;
```

Comment:
Is `_content` meant to capture from the Changeset and `content` from an ObjectProxyNode?

Reply:
Yes, that's correct.

Reply:
Sry for stringing this along but I do want to get a solution in.  I'm learning from your work to understand the issue better.

So is it best to say, not only does notifyPropertyChange not work for nested keys (this is obvious to me), but also we are notifying on the wrong object?  Meaning `notifyPropertyChange(this, key)` leads to the wrong/no expected behaviour as well?  I wouldn't have through this would have work for an ObjectProxyNode (non Ember object, etc)